wazo_chatd/plugins/config/http.py

```diff
@@ -1,17 +1,39 @@
-# Copyright 2019-2021 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2019-2024 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from __future__ import annotations
+
+import logging
+
+import jsonpatch
+from flask import request
 from xivo.auth_verifier import required_acl
 
 from wazo_chatd.auth import required_master_tenant
 from wazo_chatd.http import AuthResource
 
+from .schemas import config_patch_schema
+
 
 class ConfigResource(AuthResource):
-    def __init__(self, config):
+    def __init__(self, config) -> None:
         self._config = config
 
+    def _toggle_debug_flag(self) -> None:
+        logger = logging.getLogger()
+        logger.setLevel(logging.DEBUG) if self._config['debug'] else logger.setLevel(
+            self._config['log_level']
+        )
+
     @required_master_tenant()
     @required_acl('chatd.config.read')
-    def get(self):
+    def get(self) -> tuple[dict, int]:
+        return dict(self._config), 200
+
+    @required_master_tenant()
```

Comment:
If somebody removes this line in the future, no test will raise any error. Could you add a test for this, please?

Reply:
Test added :+1: